### PR TITLE
ci: k8s: Adapt "source ..." to the new location of gha-run.sh

### PR DESCRIPTION
**Pull request by fidencio**
This is a follow up of 2ee2cd307b001f8eab4d8192271cf0e3fec4c595, which changed the location of gha-run.sh

Fixes: #7373

### CODE REVIEW

**Review by sprt — APPROVED** [review]
LGTM, pending CI

File: tests/integration/kubernetes/gha-run.sh
Comment (fidencio) [reply tests/integration/kubernetes/gha-run.sh]: Yep, fixed, thanks!

Level of concentration Today is below than average. :-/

**Review by stevenhorsman — APPROVED** [review]
Looks good. thanks

**Comment by fidencio** [comment]
**Please**, let's wait till the k8s tests run before merging this one.

**Review by GabyCT — APPROVED** [review]

**Pull request merged**
